<!-- RC Camera launch file -->

<launch>
	<group ns="RC_Camera">
		<!-- Camera node -->
		<node pkg="rc_camera" name="Camera"  type="rc_camera"> <!--  output="screen"-->
			<param name="image_pub" value="/rcCamera/image_raw"/> 
			<param name="width" type="int" value="1920" />
			<param name="height" type="int" value="1080" />
			<param name="fps" type="int" value="5" />
			<param name="auto_focus" type="bool" value="False" />
			<param name="focus_absolute" type="int" value="0" />
			<!-- Other supported params: auto_exposure, exposure_absolute, brightness, power_line_frequency -->

			<param name="device" type="string" value="/dev/video0" />
			<param name="camera_info_url" type="string" value="/home/student/catkin_ws/src/rc_rsd/RC_Camera/camera.yaml" />
		</node>

		<!-- Image rectification node -->
		<node pkg="image_proc" type="image_proc" name="image_proc">
			<remap from="image_raw" to="/rcCamera/image_raw"/>
			<remap from="camera_info" to="/rcCamera/camera_info"/>
		</node>
		<!-- While there are no subscribers to output topics, image_proc unsubscribes from the image_raw and camera_info topics. -->
	</group>
</launch>
